### ISSUE TITLE
Add error source info to xml stdout

### ISSUE THREAD

**Issue by matz3** [JamesMGreene/qunit-reporter-junit#18]
QUnit provides a "source" information with a detailed stacktrace for failing tests.
Could this information also be provided in the resulting xml? Would be great to get the full stacktrace directly in Jenkins CI.

I could apply a workaround for this, but it would be a cleaner solution to have this out of the box.

Before including `qunit-reporter-junit.js`
```js
QUnit.log(function(data) {
	// manipulate data.message for failing tests with source info
	if (!data.result && data.source) {
		// save original error message (see QUnit.log below)
		data.___message = data.message;
		// add source info to message for detailed reporting
		data.message += '\n' + 'Source: ' + data.source;
	}
});
```
After including `qunit-reporter-junit.js`
```js
QUnit.log(function(data) {
	if (!data.result && data.source) {
		// restore original message to prevent adding the source info to the error message title
		data.message = data.___message;
		data.___message = undefined;
	}
});
```

**Comment by JamesMGreene** [comment]
Added and released in `v1.1.1`. Thanks!

**Comment by matz3** [comment]
Cool, thanks 👍
